### ISSUE TITLE
Menu fetching doesn't work

### ISSUE THREAD

**Issue by Friendly-Banana** [TUM-Dev/eat-api#251]
https://www.studierendenwerk-muenchen-oberbayern.de/mensa/speiseplan/speiseplan_422_-de.html shows dishes, but [the API](https://github.com/TUM-Dev/eat-api/blob/gh-pages/mensa-garching/2024/30.json) returns none.

**Comment by atticus-sullivan** [comment]
Yes, studierendenwerk changed some things on their website (eg including a note about water consumption and CO2 emissions for the dishes) and thereby the class `js-schedule-dish-description` doesn't seem to be available anymore ([here's](https://github.com/atticus-sullivan/signalbot/commit/7b89ef7a3f4cd0e2ef67faedb2c36dee02ed5eaf#diff-c4e782deb83840a1b835526f6380b61c8b0c55858875402311625d20ffb243dfL41) how I fixed this for me).

For the `eat-api` the relevant line should be [this](https://github.com/TUM-Dev/eat-api/blob/d4895cac82c26bcf3bc8003fb6886c9b9d25cf59/src/menu_parser.py#L309) I think (I don't really know the codebase, so I didn't implement or even test this for the `eat-api`).

Apart from using the new class everything should work like before (if you don't want to parse the new information provided on the studierendenwerk website).
